packages/commercetools-api-request-builder/src/create-service.js

```diff
@@ -92,18 +92,27 @@ export default function createService (
     // Pass some options to further configure the URI:
     // - `projectKey`: will prefix the URI with the given projectKey
     build (options: BuildOptions = {}): string {
+      // FIXME: make it an option of `createRequestBuilder`
```

Comment:
Should resolve this before merging or remove it & open an issue about it so we can track it there.